server/service/http/api.go

```diff
@@ -108,6 +120,7 @@ func (a *API) NewAPIRouter() *Router {
 	router.Get("/debug/pprof/block", a.pprofBlock)
 	router.Get("/debug/pprof/goroutine", a.pprofGoroutine)
 	router.Get("/debug/pprof/threadCreate", a.pprofThreadcreate)
+	router.Get("/debug/diagnose/shards/:"+clusterNameParam, wrap(a.diagnoseShards, true, a.forwardClient))
```

Comment:
`/debug/diagnose/:cluster/shards` is more suitable, in later we may need to check other status of one cluster, such as tables.